lib/ViewModels/FeatureInfoPanelViewModel.js

```diff
@@ -139,21 +139,25 @@ FeatureInfoPanelViewModel.prototype.showFeatures = function(features) {
         }
 
         var feature = features.features[0];
+
         if (!defined(feature.position)) {
             feature.position = features.pickPosition;
         }
-
-         that.terria.selectedFeature = feature;
+        that.terria.selectedFeature = feature;
         that.name = feature.name ? feature.name : feature.id;
-        if (defined(feature.description)) {
-            that.html = feature.description.getValue( that.terria.clock.currentTime);
+
+        if (defined(feature.properties)) {
+            var data = feature.properties.getValue(feature),
```

Comment:
Should probably pass `that.terria.clock.currentTime` instead of `feature`, right?